packages/HttpTransport/src/HttpTransport.js

```diff
@@ -0,0 +1,58 @@
+import get from 'lodash/get';
+import last from 'lodash/last';
+import Promise from 'bluebird';
+import { EventEmitter } from 'events';
+
+function getRequestId(command) {
+  return get(command, 'headers.id');
+}
+
+function getQueueName(command) {
+  return get(command, 'headers.queueName');
+}
+
+class HttpTransport {
+  constructor({server, log, sendingOnly}) {
+    this.log = log;
+    this.sendingOnly = sendingOnly;
+    this.server = server;
+    this.messageHandler = new EventEmitter();
+    this.requestHandler = new EventEmitter();
+
+    this.listenOnServer();
+  }
+
+  listenOnServer() {
+    console.log(HttpTransport.ENQUEUE_URI)
+    this.server.post(HttpTransport.ENQUEUE_URI, (req, res) => {
```

Comment:
you want to create a generic class that accepts a http server, but this is an `express` only API.

I prefer that we use `express` as the http server (with a dependency). express routing is pretty nice, so we can use it.

@sloops77 we don't want different http urls for different endpoints ?